README.md

```diff
@@ -30,3 +30,88 @@ The aims of the project are:
 3.	Calculate a ratio between public and private transport journey times 
 4.	Calculate an overall metric for the entire city
 5.	Compare the results for each city
+
+### Using VS code
+
+#### Create new branch 
+
+Initially created a branch on GitHub from the develop branch. 
+Within the terminal:
+```python 
+git fetch -p
+``` 
+This will update your local branches in line with remote branches
+Within VS code you can switch branches using the button in the bottom left corner. 
+Within the terminal you can switch branches using:
+```python
+git checkout branch_name
+```
+
+#### Moving changes from one branch to another
+It is important to ensure that you are in the correct branch before making changes however, you can move changes if needed. 
+The changes need to be added but not yet committed.
+To add the changes:
+```python
```

Comment:
Same for all the other `git..` commands. It's all command line (shell?), not Python.